app/models/order.rb

```diff
@@ -68,4 +69,13 @@ def set_payment_status
   def paid
     update(payment_status: PAID)
   end
+
+  def address
+    [city, street, building].compact.join(', ')
+  end
+
+  def geocode_address
```

Comment:
I think this method should be in the geocoded_address model also you have an association with the same name as this method so you are redefining this method logic so it won't work as expected it will always create a new GeocodedAddress